Gopkg.toml

```diff
@@ -1,6 +1,6 @@
 [[constraint]]
   name = "github.com/container-storage-interface/spec"
-  version = "~0.2.0"
+  version = "~0.3.0"
```

Comment:
CSI version 3.0 has dependency with protobuf version 1.1.0.
So we need to add protobuf dependency as well.
```
[[override]]
  name = "github.com/golang/protobuf"
  version = "1.1.0"
```

See: https://github.com/container-storage-interface/spec/blob/f7a67e709690807e0d6f7a6905c5900e338ca017/lib/go/Makefile#L61

Reply:
I didn't see the `Gopkg.lock`.
But I think the explicit protobuf version is good.